assets/src/design-system/components/search/test/search.js

```diff
@@ -0,0 +1,272 @@
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { fireEvent, waitFor, act } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { renderWithProviders } from '../../../testUtils/renderWithProviders';
+import { basicDropDownOptions } from '../../../testUtils/sampleData';
+import { Search } from '..';
+
+describe('Search <Search />', () => {
+  // Mock scrollTo
+  const scrollTo = jest.fn();
+  let nativeScrollTo;
+  beforeEach(() => {
+    nativeScrollTo = window.scrollTo;
+    window.scrollTo = scrollTo;
+  });
+
+  afterEach(() => {
+    window.scrollTo = nativeScrollTo;
+  });
+
+  jest.useFakeTimers();
+
+  it('should render a closed <Search /> menu with an input field on default', () => {
+    const { getByRole, queryAllByRole } = renderWithProviders(
+      <Search options={basicDropDownOptions} ariaInputLabel="label" />
+    );
+
+    const input = getByRole('combobox');
+    expect(input).toBeInTheDocument();
+
+    const menu = queryAllByRole('listbox');
+    expect(menu).toStrictEqual([]);
+  });
+
+  it('should show placeholder value when no selected value is found', () => {
+    const { getByPlaceholderText } = renderWithProviders(
+      <Search options={basicDropDownOptions} placeholder="select a value" />
+    );
+
+    const placeholder = getByPlaceholderText('select a value');
+    expect(placeholder).toBeInTheDocument();
+  });
+
+  it("should show selectedValue's associated label in input when selectedValue is present", () => {
+    const container = renderWithProviders(
+      <Search
+        options={basicDropDownOptions}
+        placeholder="select a value"
+        ariaInputLabel="my aria label"
+        selectedValue={basicDropDownOptions[2]}
+      />
+    );
+
+    const input = container.getByDisplayValue(basicDropDownOptions[2].label);
+    expect(input).toBeInTheDocument();
+  });
+
+  it('should allow inputValue to update outside of options in menu', async () => {
+    const container = renderWithProviders(
+      <Search
+        options={basicDropDownOptions}
+        placeholder="select a value"
+        ariaInputLabel="my label"
+      />
+    );
+
+    const input = container.getByPlaceholderText('select a value');
+
+    act(() => {
+      fireEvent.change(input, { target: { value: 'bruce wayne' } });
+    });
+
+    await waitFor(() => {
+      expect(container.getByDisplayValue('bruce wayne')).toBeInTheDocument();
+    });
+
+    expect(input).toBeInTheDocument();
+  });
+
+  it('should show <Search /> menu when input has more than 1 character', () => {
```

Comment:
it's one or more right?
```suggestion
  it('should show <Search /> menu when input has 1 or more characters', () => {
```